doc/releases/changelog-dev.md

```diff
@@ -146,6 +146,36 @@
   The adjoint method requires memory for 4 independent state vectors, which corresponds roughly
   to storing a state vector of a system with 2 additional qubits.
 
+* A new gradient transform `qml.gradients.param_shift_hessian` has been added to directly compute
```

Comment:
Should the added `hessian_transform` be mentioned here as well as an addition? :) 
And maybe in conjunction with that:
```suggestion
* A new Hessian transform `qml.gradients.param_shift_hessian` has been added to directly compute
```

Reply:
Thanks for the suggestion, I've adjusted the changelog accordingly :)